WebP.xcodeproj/project.pbxproj

```diff
@@ -950,12 +950,48 @@
 		5B42DE8F1A63D1930025C1EA /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CLANG_WARN_BOOL_CONVERSION = YES;
```

Comment:
The WebP project is based on config files which include the standard Sketch config files (I know, I know). Shouldn't these settings just come straight from those configs rather than being set in the project?